examples/multisig_plain/lib.rs

```diff
@@ -307,13 +308,19 @@ mod multisig_plain {
         /// };
         ///
         /// // submit the transaction for confirmation
-        /// let mut submit = CallParams::eval(wallet_id, Selector::from_str("submit_transaction"));
-        /// let (id, _)  = submit.push_arg(&transaction)
+        /// let mut submit = CallParams::<Env, _, _>::eval(
+        ///     wallet_id,
+        ///     Selector::from_str("submit_transaction")
```

Comment:
Can you replace this call with `Selector::new([u8; 4])`? We are about to remove `Selector::from_str` since computing the selector is actually an ink! implementation detail exposed by the metadata. So what you could do instead is to compute it manually on your machine and simply insert the resulting bytes. The hash should be BLAKE-2 and the input to it are simply the string as bytes. Especially for the upcoming trait support we need to extend or change this system which is why we do not want to expose some stable interface for it (`Selector::from_str`).